### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by rtio**
## What/why

- This pull request improves the readme file by adding information about the:
  -  the project;
  - Makefile commands;
  - a few other relevant things.

## How to test

- Please read the file and look for typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,124 @@
...
+
+You can install the package via Composer:
+
+```bash
 composer require tumblr/chorus-timekeeper
 ```
+
+## Development Workflow
+
+### Basic Usage
```
Comment (sanmai) [inline README.md]:
```suggestion
## Basic Usage
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,124 @@
...
+
+## Installation
+
+You can install the package via Composer:
+
+```bash
 composer require tumblr/chorus-timekeeper
 ```
+
+## Development Workflow
```
Comment (sanmai) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,124 @@
...
+echo $timekeeper->getCurrentUnixTime();  // Outputs the current unix epoch time.
+
+// Using the FakeTimeKeeper
+$fakeTimeKeeper = new Tumblr\Chorus\FakeTimeKeeper(1628700000);  // Set a specific unix time.
+
+echo $fakeTimeKeeper->getCurrentUnixTime();  // Outputs 1628700000.
+
+```
+
+The project has a Makefile to simplify development processes. You can use various make commands to run tests, validate code standards, and more. Here's a quick overview:
```
Comment (sanmai) [inline README.md:41]:
```suggestion
## Development Workflow

The project has a Makefile to simplify development processes. You can use various make commands to run tests, validate code standards, and more. Here's a quick overview:
```

**Review by sanmai — APPROVED** [review]
💜

File: README.md [hunk 4]
```diff
@@ -1,7 +1,125 @@
...
+
+echo $fakeTimeKeeper->getCurrentUnixTime();  // Outputs 1628700000.
+
+```
+
+## Development Workflow
+
+The project has a Makefile to simplify development processes. You can use various make commands to run tests, validate code standards, and more. Here's a quick overview:
+
+### 1. Prerequisites Setup
```
Comment (sanmai) [inline README.md:43]: I would usually say: call `make` and it does all you need. What's your take?

File: README.md [hunk 5]
```diff
@@ -1,7 +1,125 @@
...
+4. **Describe Your Changes**: Clearly describe what you've changed and include testing instructions.
+5. **Breaking Changes Alert**: If your contribution introduces breaking changes, please highlight them. Any modification that disrupts the current workflow, like changing a namespace or functionality, is a breaking change.
+
+Remember, if you're planning a significant refactor or introducing breaking changes, your PR might not be approved.
+
+Once your PR is approved, it will be merged, and a new version will be released.
+
+## License
+
+This project is licensed under the [GNU GENERAL PUBLIC LICENSE](LICENSE).
```
Comment (sanmai) [inline README.md]:
```suggestion
This project is licensed under the [GNU General Public License v2.0](LICENSE).
```

**Review by sanmai — APPROVED** [review]
Let's go!

File: README.md (hunk 4 above)
Comment (rtio) [reply README.md:43]: It sounds good, but I generally like to make it as descriptive as possible. I usually keep that mentality: if the reader has never heard of the Makefile, how can I give them a hint?

I’ll certainly reconsider the balance between being descriptive and being redundant.

**Pull request merged**